Header header
Bbox bbox
bool locked
float32 distance
